app/views/hubstats/deploys/show.html.erb

```diff
@@ -4,7 +4,7 @@
     <!-- Title with the repo name and git revision sha -->
     <h1 class="title text-center">
       <%= link_to @deploy.repo.name.titleize, repo_path(@deploy.repo.name) %> /
-      <%= @deploy.git_revision.titleize %>
+      <%= @deploy.git_revision %>
```

Comment:
![screen shot 2016-01-12 at 3 35 23 pm](https://cloud.githubusercontent.com/assets/7562793/12277665/2d02faca-b942-11e5-97ad-f3c10cd2952d.png)

In reality, the commit sha of this is `e00de46`, not `E00de46`.